R-package/R/create-release-files.R

```diff
@@ -0,0 +1,411 @@
+source(file.path(getwd(), "R", "odm-dictionary-file.R"))
+source(file.path(getwd(), "R", "files.R"))
+#' Create release files
+#'
+#' Creates release files given the user OSF link and auth token.
+#'
+#' @param OSF_LINK_RELEASE link to the dictionary stored on OSF used for updating.
+#' @param OSF_LINK_PAST_RELEASE link to the dictionary stored on OSF used for removal of previous release files.
+#' @param OSF_TOKEN OSF auth token used for modifying OSF directories
+#' @param dictionary_path optional string containing path to the dictionary directory. When provided no dictionary is downloaded.
+#' @param past_dictionary_path optional string containing path to the previous release dictionary directory. When provided no dictionary is downloaded.
+#'
+#' @export
+create_release_files <-
+  function(OSF_LINK_RELEASE,
+           OSF_LINK_PAST_RELEASE,
+           OSF_TOKEN,
+           dictionary_path = NULL,
+           past_dictionary_path = NULL) {
+    # Setup logging
+    # Remove previous log
+    file.remove(odm_dictionary$log_path)
```

Comment:
I thought it would be easier to review a log file rather then console logs. But it could be a toggle for local building to use log file but using only console for production.

Reply:
During meeting discussion, it was decided to use the log and attach it as an artifact to the github action for later download and review. https://docs.github.com/en/actions/managing-workflow-runs/downloading-workflow-artifacts